local/kaltura/version.php

```diff
@@ -25,8 +25,45 @@
     die('Direct access to this script is forbidden.');
 }
 
-$plugin->version = 2021051700;
+$plugin->version = 2021051701;
 $plugin->component = 'local_kaltura';
-$plugin->release = 'Kaltura release 4.3.0';
+$plugin->release = 'Kaltura release 4.3.1';
 $plugin->requires = 2018120300;
 $plugin->maturity = MATURITY_STABLE;
+
+global $DB;
+
+$localKalturaPluginVersionRecords = $DB->get_records_select('config_plugins', "plugin = 'local_kaltura' AND name = 'version'");
+
+$kalturaPluginVersion = "";
+if ($localKalturaPluginVersionRecords) {
+    foreach ($localKalturaPluginVersionRecords as $key => $localKalturaPluginVersionRecord) {
+        $kalturaPluginVersion = $localKalturaPluginVersionRecord->value;
+        break;
+    }
+}
+
+if ($kalturaPluginVersion == 20210620311) {
+    $pluginsRecords = $DB->get_records_select('config_plugins', "name = 'version' AND value = '$kalturaPluginVersion'");
+
+    foreach ($pluginsRecords as $record) {
+        $record->value = 2021051700;
+        $DB->update_record('config_plugins', $record);
+    }
+}
+else if ($kalturaPluginVersion == 20201215310 || $kalturaPluginVersion == 20210620310) {
+    $pluginsRecords = $DB->get_records_select('config_plugins', "name = 'version' AND value = '$kalturaPluginVersion'");
```

Comment:
these blocks seem identical except for the target record value being set.
you can probably do an if/elseif/elseif block to only set a variable ```$resetDbVersion``` to the desired value, and then write once the line to get records and the loop to update records.